### PR TITLE
Redis v4 in Dockerfile

### PR DESCRIPTION
**Pull request by oleurud**
We need Redis v4 for rate limits feature.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -51,6 +51,14 @@ RUN set -ex \
     && make install \
     && cd .. \
     && rm -rf postgis-2.4.0 \
+    && wget http://download.redis.io/redis-stable.tar.gz \
```
Comment (davidmanzanares) [inline Dockerfile]: Is this aiming to version 4 or to the latest stable version?

Comment (oleurud) [reply Dockerfile]: I am going to change it to an specific version

Comment (davidmanzanares) [reply Dockerfile]: :+1:

File: NEWS.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 2018-mm-dd
 
 Announcements:
-
+ - Adding Redis v4 ikn Dockerfile
```
Comment (davidmanzanares) [inline NEWS.md]: Typo `in` :slightly_smiling_face:

**Pull request merged**